SConstruct

```diff
@@ -553,9 +555,11 @@ config_options = [
            conjunction with "prefix='/usr/local'". 'compact' puts all installed
            files in the subdirectory defined by 'prefix'. This layout is best
            with a prefix like '/opt/cantera'. 'debian' installs to the stage
-           directory in a layout used for generating Debian packages.""",
+           directory in a layout used for generating Debian packages. If the Python
+           executable found during compilation is managed by 'conda', the layout
+           will default to 'conda' irrespective of operating system.""",
```

Comment:
I think this also needs to provide some description of what the `conda` layout does, that is, that it installs Cantera into the active conda environment.